cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/resource/AbstractOperations.java

```diff
@@ -215,18 +222,19 @@ public T get(String namespace, String name) {
         return fut;
     }
 
+    /**
+     * Check if a resource is in the Ready state
+     *
+     * @param namespace The namespace.
+     * @param name  The resource name.
+     * @return  if it's in the Ready state
```

Comment:
For a method like this, where the description basically says the same thing as the `@return`, I think it's better to just specify once what the return value is. I tend to use the description for that and omit the `@return`. We should seek to be consistent. I don't mind changing to something else, but we need to agree what our preferred style is.